bindings/dd/register_matrix_dds.cpp

```
@@ -13,77 +13,99 @@
+      m, "MatrixDD", "A class representing a matrix decision diagram (DD).");
+
+  mat.def("is_terminal", &dd::mEdge::isTerminal,
+          "Check if the DD is a terminal node.");
+  mat.def("is_zero_terminal", &dd::mEdge::isZeroTerminal,
+          "Check if the DD is a zero terminal node.");
 
-  mat.def("is_terminal", &dd::mEdge::isTerminal);
-  mat.def("is_zero_terminal", &dd::mEdge::isZeroTerminal);
   mat.def("is_identity", &dd::mEdge::isIdentity<>,
-          "up_to_global_phase"_a = true);
+          "up_to_global_phase"_a = true,
```

Cppcheck (2.18.0)

[error] Boolean value assigned to pointer.

(assignBoolToPointer)